pcm-player.js

```diff
@@ -1,8 +1,8 @@
-function PCMPlayer(option) {
-    this.init(option);
+function PCMPlayer(option, callback) {
+    this.init(option, callback);
 }
 
-PCMPlayer.prototype.init = function(option) {
+PCMPlayer.prototype.init = function(option, callback) {
```

Comment:
I'd name it `onendedCallback` in case you would like to handle some other events in the future